mods/testing.py

```diff
@@ -296,9 +301,7 @@ def prepare_project_hook(self, request, project):
                                    "class": "info",
                                    "content_html": self.build_config_html(request,
                                                                           project)})
```

Comment:
We could use format_html in self.build_config_html() and drop the "safe" filter in the template later, right?

Reply:
Yes, exactly.